.happy/terraform/modules/sfn/main.tf

```diff
@@ -195,6 +195,69 @@ resource "aws_sfn_state_machine" "state_machine" {
 EOF
 }
 
+
+resource "aws_sfn_state_machine" "state_machine_seurat" {
+  name     = "dp-${var.deployment_stage}-${var.custom_stack_name}-seurat-sfn"
+  role_arn = var.role_arn
+
+  definition = <<EOF
+{
+  "StartAt": "Seurat",
+  "States": {
+    "Seurat": {
+      "Type": "Task",
+      "End": true,
+      "Resource": "arn:aws:states:::batch:submitJob.sync",
+      "Parameters": {
+        "JobDefinition": "${var.job_definition_arn}",
+        "JobName": "cxg",
+        "JobQueue": "${var.job_queue_arn}",
+        "ContainerOverrides": {
+          "Environment": [
+            {
+              "Name": "DATASET_ID",
+              "Value.$": "$.dataset_uuid"
+            },
+            {
+              "Name": "STEP_NAME",
+              "Value": "seurat"
+            }
+          ]
+        }
+      },
+      "TimeoutSeconds": 36000,
+      "Retry": [
```

Comment:
We probably don't need retries on conversion failures.

Reply:
Yeah, gonna remove them.

Reply:
As noted in the [infra PR](https://github.com/chanzuckerberg/single-cell-infra/pull/461/files#r759431351), we should keep the retries.